frontend/src/js/tooltip/AdditionalInfoHoverable.tsx

```diff
@@ -27,18 +27,24 @@ const AdditionalInfoHoverable = ({
   node,
   className,
   children,
+  parent,
 }: {
   children: ReactNode;
   node: ConceptT;
   className?: string;
+  parent?: string;
```

Comment:
And this is the parentLabel now, right? So how about calling it
```suggestion
  parentLabel?: string;
```